toyota_prius_2017_pt_generated.dbc

```diff
@@ -167,6 +167,15 @@ BO_ 951 ESP_CONTROL: 8 ESP
  SG_ BRAKE_HOLD_ENABLED : 33|1@1+ (1,0) [0|1] "" XXX
  SG_ BRAKE_HOLD_ACTIVE : 36|1@0+ (1,0) [0|1] "" XXX
 
+BO_ 956 GEAR_PACKET: 8 XXX
```

Comment:
Need to rename the duplication, Prius also has `GEAR_PACKET_HYBRID` on `295`

Reply:
I reverted that so that we only change the nodsu DBCs in this PR. As adeeb said, it'll make getting it merged much easier if we only have to verify a couple of TSS2 hybrids rather than all. Next PR can be the rest of the cars